### DISCUSSION TITLE
Ergonomics for JsFunction.call?

### DISCUSSION THREAD

**Discussion by safasofuoglu** [napi-rs/napi-rs#636] [Q&A]
Since `JsFunction.call` accepts an array, the only way I was able to call a `JsFunction` with different types of arguments is calling `into_unknown()` for all arguments and passing that `JsUnknown` array to `call`. Is this the recommended way? Have you considered writing a vararg macro?

**Comment by Brooooooklyn** [comment]
> Have you considered writing a vararg macro?

Oh yes, I will try it in this month.
